code/game/objects/items/devices/temperature_control_device.dm

```diff
@@ -0,0 +1,26 @@
+/obj/item/tcd
+	name = "temperature control device"
+	desc = "A handheld device that can cool or heat a container."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "tcd-hot"
+	item_state = "tcd-hot"
+	w_class = WEIGHT_CLASS_SMALL
+	slot_flags = SLOT_BELT
+	materials = list(MAT_METAL=200, MAT_GLASS=200)
+	force = 2
+	throwforce = 0
+	var/actingtemp = 500
+	var/hot_temp = 500
+	var/cold_temp = 250
```

Comment:
The numbers here don't match those in the PR description (neither heating nor cooling, but i guess the former weren't meant to be exact anyway so it's fine).
Please realign (at least the cooling ones).

Reply:
Updated the PR description to be the same as the values assigned.